libs/src/api/serviceProvider.js

```diff
@@ -40,12 +40,22 @@ class ServiceProvider extends Base {
     }
 
     // Time requests and get version info
-    const timings = await Utils.timeRequests(
-      creatorNodes.map(node => ({
-        id: node.endpoint,
-        url: `${node.endpoint}/version`
-      }))
-    )
+    let timings = (await Promise.all(
+      creatorNodes.map(async node => {
+        let resp
+        const request = {
+          id: node.endpoint,
+          url: `${node.endpoint}/version`
+        }
+        try {
+          resp = await Utils.timeRequest(request)
+        } catch (e) {
+          resp = false
+        }
+        return resp
+      })
+    )).filter(Boolean)
+    timings = timings.sort((a, b) => a.millis - b.millis)
```

Comment:
basically re-wrote the `Utils.timeRequests()` function here, with an added try-catch on each `timeRequest()` call because without that if any of the requests failed the whole thing would throw and error out. With this code change, we can implicitly use this to also filter out unhealthy services

Reply:
was there a reason we didn't just want to fix it inside timeRequests? iiuc, this change is just better?